admin/sql/updates/2019-07-09-add-recommendation-table.sql

```diff
@@ -0,0 +1,48 @@
+BEGIN;
+
+CREATE SCHEMA recommendation;
+
+CREATE TABLE recommendation.cf_recording (
+  id                  SERIAL, -- PK
+  user_id             INTEGER NOT NULL, --FK to "user".id
+  msid                UUID NOT NULL,
+  type                recording_type,
+  created             TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+);
+
+CREATE TABLE recommendation.recommender (
+  id                  SERIAL, --PK
+  repository          TEXT NOT NULL,
+  author_email        TEXT NOT NULL,
+  name                TEXT NOT NULL,
+  created             TIMESTAMP WITH TIME ZONE DEFAULT NOW()
+);
+
+CREATE TABLE recommendation.cf_recording_recommender_join(
+  last_used           TIMESTAMP WITH TIME ZONE DEFAULT NOW() NOT NULL,
+  cf_recording_id     INTEGER, --FK to recommendation.cf_recording.id
```

Comment:
Since we're going to change the tracks periodically, we could not delete them from cf_recording if we had this FK. We should change this to: "recording_msid UUID NOT NULL"